sql/scriptdev2/scriptdev2.sql

```diff
@@ -659,6 +659,7 @@ UPDATE instance_template SET ScriptName='instance_ramparts' WHERE map=543;
 
 /* SHATTERED HALLS */
 /* Nethekurse and his spawned shadowfissure */
+UPDATE creature_template SET AIName="",  ScriptName = "mob_shattered_hand_legionnair" WHERE Entry="16700";
```

Comment:
```
-- ============================================================
-- Remove All Creature Template Entries That Conflict With ACID
-- ============================================================
UPDATE creature_template SET AIName= '' WHERE AIName= 'EventAI';
UPDATE creature_template SET ScriptName= '' WHERE ScriptName= 'generic_creature';


-- ================================================
-- Define ACID Full Release Script Creature Entries
-- ================================================
UPDATE creature_template INNER JOIN creature_ai_scripts ON creature_template.entry = creature_ai_scripts.creature_id SET AIName='EventAI';
```

https://raw.githubusercontent.com/cmangos/tbc-db/master/ACID/acid_tbc.sql

no need to set AIName